lsp-methods.el

```diff
@@ -940,7 +940,17 @@ https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#co
       :annotation-function #'lsp--annotate
       :display-sort-function #'lsp--sort-completions)))
 
-;;; TODO: implement completionItem/resolve
+(defun lsp--resolve-completion (comp)
+  (lsp--cur-workspace-check)
+  (lsp--send-changes lsp--cur-workspace)
+  (if (gethash "resolveProvider" (lsp--capability "completionProvider"))
+    (progn
+      (when (null comp)
```

Comment:
I would make it fail if `resolveProvider` is nil. Reasons:

 * It's more consistent between the 2 cases
 * It's an error the caller should take care of. Early throwing is better than silently swallowing it.